ptf_tests/tests/l3_dpdk_port_flapping_with_link_ports.py

```diff
@@ -0,0 +1,204 @@
+           vm_cmd_list.append(globals()["vm"+str(vm_id+1)+"_command_list"])
+           vm_id+=1 
+
+        # configuring VMs
+        for i in range(len(self.conn_obj_list)):
+            print (f"Configuring VM{i}....")
+            test_utils.configure_vm(self.conn_obj_list[i], vm_cmd_list[i])
+       
+        port_list = self.config_data['port_list']
+        port_list[0] = test_utils.get_port_name_from_pci_bdf(self.capture_port)
+        phy_interface1 = port_list[0]
+        port_ids = test_utils.add_port_to_dataplane(port_list)
+        print(phy_interface1)
+
+        # Now bring PHY interface down
+        print('Bring PHY1 interface down...')
+        portconfig_obj = port_config.PortConfig()
+        portconfig_obj.Ip.iplink_enable_disable_link(phy_interface1, status_to_change='down')
+
+
+        # Now change Mac address 
+        print('Change Mac of PHY1 interface...')
+        portconfig_obj = port_config.PortConfig()
+        portconfig_obj.Ip.ip_link_set_mac(phy_interface1, self.config_data['vm'][0]['mac_remote'])
+
+        # Now bring PHY interface up
+        print('Bring PHY1 interface up...')
+        portconfig_obj = port_config.PortConfig()
+        portconfig_obj.Ip.iplink_enable_disable_link(phy_interface1, status_to_change='up')
+
+
+        # Now Give Ip address
+        print('Give Ip address on  PHY1 interface...')
+        portconfig_obj = port_config.PortConfig()
+        portconfig_obj.Ip.ipaddr_ipv4_set(phy_interface1, self.config_data['port'][1]['ip_address'])
+
+
+        # Now bring PHY interface up
+        print('Bring PHY1 interface up...')
+        portconfig_obj = port_config.PortConfig()
+        portconfig_obj.Ip.iplink_enable_disable_link(phy_interface1, status_to_change='up')
+
+        # Now Add neighbour entry 
+        print('Add neigh entry on  PHY1 interface...')
+        portconfig_obj = port_config.PortConfig()
+        portconfig_obj.Ip.ip_neigh_add(phy_interface1, self.config_data['port'][1]['remote_ip'], self.config_data['port'][1]['mac_local'])
+
+        # ping test between VM to link port
+        print("------------------------------")
+        print("Ping Test from VM to Link port")
+        print("------------------------------")
+        result = test_utils.vm_to_vm_ping_test(self.conn_obj_list[0], self.config_data['vm'][0]['remote_ip'])
+        if not result:
+            self.result.addFailure(self, sys.exc_info())
+            print("FAIL: Ping test failed for VM0")
+
+        print("------------------------------")
+        print("Ping Test from Link port to VM")
+        print("------------------------------")
+        #Ping from link port to vhost
+        pingcmd = f"ping -I {phy_interface1}  {self.config_data['port'][1]['remote_ip']} -c {self.config_data['num']}"
```

Comment:
local_ping expects *args, so instead of sending a string try a list (good to do, wont break anything). 
pingcmd = f"ping -I {phy_interface1}  {self.config_data['port'][1]['remote_ip']} -c {self.config_data['num']}".split()